### PR TITLE
final commit on my hobby

### PR DESCRIPTION
**Pull request by Clementify**
I am done with my Lesson 1 and 2 task5, kindly review @ChidiPatrick and @Promise-J.

### CODE REVIEW

File: html/myhobby.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                the 20th century led to mass production and export of sewn
+                objects, but hand sewing is still practiced around the world.
+                Fine hand sewing is a characteristic of high-quality tailoring,
+                haute couture fashion, and custom dressmaking, and is pursued by
+                both textile artists and hobbyists as a means of creative
+                expression. The first known use of the word
+                "sewing" was in the 14th century
+            </p>
+
+            <img src="/image/Tailoring.jpg" alt="tailoring">
```
Comment (ChidiPatrick) [inline html/myhobby.html]: close your img tag

**Review by ChidiPatrick — CHANGES_REQUESTED** [review]
Do the correction and let me know

File: html/myhobby.html (hunk 1 above)
Comment (Clementify) [reply html/myhobby.html]: Alright sir,

**Comment by Clementify** [comment]
I have closed it sir. Kindly review @ChidiPatrick and @Promise-J

**Review by ChidiPatrick — APPROVED** [review]
Great work!
You nailed it!

**Comment by Clementify** [comment]
THank you sir.

**Pull request merged**